eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -148,6 +148,12 @@
           "new": "(class|interface|enum) com\\.azure\\.perf\\.test\\.core\\..*",
           "justification": "perf-test classes are allowed to be exposed."
         },
+        {
+          "regex": true,
+          "newArchive": "com\\.azure:azure-core:.*",
+          "newArchiveRole": "supplementary",
```

Comment:
Did we test if changes in core when it's "primary" role are still getting caught ?

Reply:
Pushed an update as this wasn't doing exactly what I expected. Now this is done through a custom RevApi transform